pkg/machine/hyperv/machine.go

```diff
@@ -67,6 +67,31 @@ type HyperVMachine struct {
 	LastUp time.Time
 }
 
+func addSSHConnectionsToPodmanSocket(m *HyperVMachine, opts machine.InitOptions) error {
```

Comment:
We can turn it into a method (with comment) instead:
```golang
func (m *HyperVMachine) add... error {}
```

Reply:
Should be all set now